QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -6,21 +6,71 @@ class FunctionalTests_ItSpec: QuickSpec {
     override func spec() {
         var exampleMetadata: ExampleMetadata?
         beforeEach { metadata in exampleMetadata = metadata }
-
+        
         it("") {
             expect(exampleMetadata!.example.name).to(equal(""))
         }
-
+        
         it("has a description with セレクター名に使えない文字が入っている 👊💥") {
             let name = "has a description with セレクター名に使えない文字が入っている 👊💥"
             expect(exampleMetadata!.example.name).to(equal(name))
         }
+        
+        describe("error handling when misusing ordering") {
+            it("wraps another 'it' that will...") {
```

Comment:
nit: Rather than use ellipses, I'd phrase this as `"an it"` `"a nested it throws an error"`.

I don't have a strong opinion either way, but note that the test selector for ellipses will be: `wraps_another__it__that_will_______throw_an_error`--each `.` is turned into an `_`.

Anyway, no strong opinions here, just food for thought! :cherries: